src/symbols/Node/Node.tsx

```diff
@@ -234,13 +234,21 @@ export const Node: FC<Partial<NodeProps>> = ({
         event.srcElement = nodeRef.current;
 
         canvas.onDragEnd(event, coords, node, port);
-        onDragEnd(event, coords, node, port);
+        onDragEnd?.(event, coords, node, port);
         setDragging(false);
         setDragCursor(null);
       }
     }
   });
 
+  useEffect(() => {
+    if (enteredNode?.id === properties.id) {
+      setIsLinkable(checkNodeLinkable(properties, enteredNode, canLinkNode));
+    }
+
+    return () => setIsLinkable(true);
```

Comment:
should this be returning a function, or just `setIstLinkable(true)`

Reply:
IsLinkable - boolean, it's callback for cleaning after node unmounted, so just a 'true'

Reply:
good to know!